src/service/VerifyService.ts

```diff
@@ -149,8 +157,14 @@ export class VerifyService {
             };
         }
     }
+    shouldRun(lines: ReportLine[]): Promise<boolean> {
+        const dockerIsFine = !!lines.find((l) => l.header === 'Docker Version' && !l.recommendation);
+        return Promise.resolve(dockerIsFine);
```

Comment:
can you explain when you wouldn't want to check docker?

Reply:
When docker is not installed (previous test), checking that docker can run as regular user is ignored